src/uwtools/config/tools.py

```diff
@@ -16,25 +16,23 @@
 
 
 def compare_configs(
-    config_a_path: DefinitePath,
-    config_a_format: str,
-    config_b_path: DefinitePath,
-    config_b_format: str,
+    config_1_path: DefinitePath,
+    config_2_path: DefinitePath,
+    config_1_format: Optional[str] = None,
+    config_2_format: Optional[str] = None,
 ) -> bool:
     """
-    Compare two config files.
-
-    :param config_a_path: Path to first config file.
-    :param config_a_format: Format of first config file.
-    :param config_b_path: Path to second config file.
-    :param config_b_format: Format of second config file.
-    :return: False if config files had differences, otherwise True.
+    NB: This docstring is dynamically replaced: See compare_configs.__doc__ definition below.
     """
-
-    cfg_a = format_to_config(config_a_format)(config_a_path)
-    cfg_b = format_to_config(config_b_format)(config_b_path)
-    log.info("- %s", config_a_path)
-    log.info("+ %s", config_b_path)
+    config_1_format = _ensure_format("1st config file", config_1_format, config_1_path)
+    config_2_format = _ensure_format("2nd config file", config_2_format, config_2_path)
+    if config_1_format != config_2_format:
+        log.error("Formats do not match: %s vs %s", config_1_format, config_2_format)
+        return False
+    cfg_a = format_to_config(config_1_format)(config_1_path)
+    cfg_b = format_to_config(config_2_format)(config_2_path)
```

Comment:
You have `cfg_a` and cfg_b` here, did you want to standardize again and update to `cfg_1` and `cfg_2`?

Reply:
Good catch, fixing...